domains/slainbot.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "KhongPhaiHoangQuangHuy",
+           "email": "khongphaihoangquanghuy@gmail.com",
+           "discord": "181064730075463680"
+        },
+    
+        "record": {
+            "CNAME": "khongphaihoangquanghuy.github.io"
```

Comment:
I don't see any content on this or the original URL you sent. Are you able to send a preview?